src/Elmah.Io.AspNetCore/MessageShipper.cs

```diff
@@ -72,9 +56,10 @@ public static void Ship(string title, HttpContext context, ElmahIoSettings setti
                 return;
             }
 
-            var elmahioApi = ElmahioAPI.Create(apiKey);
+            var elmahioApi = new ElmahioAPI(new ApiKeyCredentials(apiKey));
```

Comment:
I don’t really have an idea on how ElmahIO works or how it is supposed to work, and I also haven’t looked at the rest of the code… but should you really create a new object on every logging call?

Wouldn’t it make more sense to have a single instance that is created on application startup which is then reused in all subsequent logging calls? That way, you could also move a lot of logic out of these static classes.